packages/vue/public/assets/storybook/SfFooter/googleplus.svg

```diff
@@ -0,0 +1,9 @@
jkc0eTikQIECBAgAABAgQmI5BZvzeZZCVKgMDkBbLcEDf5JiqAAAECBAgQIECAAAECBAgQaELAQFwTbZxHERmLa+dRqSoJELiEgP+W/SVQfEWAAAECBAgQIHCEQObvHrHCYwIECHQnkPX87oKJRIAAAQIECBAgQIAAAQIECBAgsK6Agbh15ezbgsDy6i0c6kgCBEYgUOG/ZT+CNkiBAAECBAgQIDA5ges+ctc9FfHtySUuYQIEJiqQl080cWkTIECAAAECBAgQIECAAAECBJoSMBDXVDtbLyavb71C9REgcJBAveSgJ74nQIAAAQIECBAgcKhA1e2HPveQAAECHQo88r7ThuI69BSKAAECBAgQIECAAAECBAgQILCOgIG4ddTs2YpARXpl6lbkHUpg+wKZ+byK8O+s7bdCBgQIECBAgACByQlk1Ccml7SECRCYsoCBuCl3T+4ECBAgQIAAAQIECBAgQIBAEwKGC5po42yKuGY2lSqUAIFnCTx401tf+qwvfUGAAAECBAgQIEDgCIHKhYG4I4w8JkCgO4EnIgzEdccpEgECBAgQIECAAAECBAgQIEBgLQEDcWux2TS0QH0gdjLiFUOf6zwCBMYjsLcfLxpPNjIhQIAAAQIECBCYisD1j112T1Q8OZV85UmAwLQFdpYnrph2BbInQIAAAQIECBAgQIAAAQIECExfwEDc9Hs4iwrOffstr55FoYokQOBAgd3F7osPfOgBAQIECBAgQIAAgQME8rbb9irqrgMe+5oAAQKdCuxFuSGuU1HBCBAgQIAAAQIECBAgQIAAAQKrCxiIW93Mjm0I5M7V2zjWmQQIjEdgfydeOJ5sZEKAAAECBAgQIDAlgcy8bUr5ypUAgekKLAzETbd5MidAgAABAgQIECBAgAABAgSaETAQ10wrGy9ksXhV4xUqjwCBowRq+ZKjlnhOgAABAgQIECBA4FICFcv/71Lf+44AAQJdC2TteGVq16jiESBAgAABAgQIECBAgAABAgRWFDAQtyKY5dsRWC7jFds52akECIxFIGvxorHkIg8CBAgQIECAAIFpCew8duquqtibVtayJUBgigLLWBqIm2Lj5EyAAAECBAgQIECAAAECBAg0JWAgrql2tltMZhmIa7e9KiNwTIF68TEXWkaAAAECBAgQIEDgAoFrb7vtyci454IvfSBAgEAPAhlxeQ9hhSRAgAABAgQIECBAgAABAgQIEFhBwEDcCliWbk+gIl++vdOdTIDAGAQqww1xY2iEHAgQIECAAAECExXIik9MNHVpEyAwIYGKdEPchPolVQIECBAgQIAAAQIECBAgQKBNAQNxbfa1uaoyvDK1uaYqiMCKAlnxwhW3WE6AAAECBAgQIEDgBwKLiP/4gw/+QIAAgb4EMp/TV2hxCRAgQIAAAQIECBAgQIAAAQIEjidgIO54TlZtWaCi3BC35R44nsD2BfJl289BBgQIECBAgAABAlMVuOLxb90+1dzlTYDAhARqaSBuQu2SKgECBAgQIECAAAECBAgQINCmgIG4NvvaVFUVkRnxyqaKUgwBAisLVNYrVt5kAwECBAgQIECAAIHvCbz0ts99Jyo+C4QAAQK9CmSc6jW+4AQIECBAgAABAgQIECBAgAABAkcKGIg7ksiCbQuce/8NL418ZibO/xAgMGeBrHjFMwOyczZQOwECBAgQIECAwKYCdeemEewnQIDAoQLllamH+nhIgAABAgQIECBAgAABAgQIEBhAwEDcAMiO2ExgcX7hVqjNCO0m0IZAZt7/rne8pI1iVEGAAAECBAgQILAdgbxjO+c6lQCBuQhkhFemzqXZ6iRAgAABAgQIECBAgAABAgRGK2AgbrStkdj3BfYX+wbivo/hnwRmLrCze95/Hsz8N6B8AgQIECBAgMAmAhkG4jbxs5cAgaMFlumGuKOVrCBAgAABAgQIECBAgAABAgQI9CtgIK5fX9E7EMhavLyDMEIQINCAwH7t+M+DBvqoBAIECBAgQIDAtgSuu/nOhyrqS9s637kECMxAoNwQN4MuK5EAAQIECBAgQIAAAQIECBAYuYCBuJE3SHoRlelGKD8EAgS+K5CLNBDnt0CAAAECBAgQILCRQEbeuVEAmwkQIHCoQHll6qE+HhIgQIAAAQIECBAgQIAAAQIE+hcwENe/sRM2FagwELepof0EGhHIKANxjfRSGQQIECBAgACBrQnU8o6tne1gAgTaF0g3xLXfZBUSIECAAAECBAgQIECAAAECYxcwEDf2DskvKg3A+BkQIPB3AlVuiPNbIECAAAECBAgQ2FSgDMRtSmg/AQIHCmSlG+IO1PGAAAECBAgQIECAAAECBAgQIDCMgIG4YZydsoFAllembsBnK4HGBAzINtZQ5RAgQIAAAQIEBhe47uZ7Ph1RTwx+sAMJEJiLgIG4uXRanQQIECBAgAABAgQIECBAgMBoBQzEjbY1EvuBQMZLf/BnfyBAYNYClW6Im/UPQPEECBAgQIAAgQ4EMmJZFZ/sIJQQBAgQeLZAloG4Z6v4hgABAgQIECBAgAABAgQIECAwqICBuEG5HbaOQEW8cJ199hAg0J5Alhvi2uuqiggQIECAAAECwwssMm8f/lQnEiAwC4HKk7OoU5EECBAgQIAAAQIECBAgQIAAgRELGIgbcXOk9ncCGXElCwIECHxXIPMaEgQIECBAgAABAgQ2FVgu665NY9hPgACBSwlU1vJS3/uOAAECBAgQIECAAAECBAgQIEBgOAEDccNZO2kNgQd/8fTL19hmCwECDQs88P4bfrzh8pRGgAABAgQIECAwgECeP3/nAMc4ggCBGQpkxN4My1YyAQIECBAgQIAAAQIECBAgQGBUAgbiRtUOyVwssLd72Ysu/s5nAgTmLVBP71w1bwHVEyBAgAABAgQIbCpw/e/e+82I+sKmcewnQIDAxQIVaSDuYhSfCRAgQIAAAQIECBAgQIAAAQIDCxiIGxjccasJLCpeuNoOqwkQaF
```

Comment:
@justyna-13 please check files after SVG export from Figma. In this case, Figma converts image to base64. Please use the old svg icon. You can remove Google + because Google disable it :)